manticore/core/state.py

```diff
@@ -184,8 +269,22 @@ def constraints(self, constraints):
         self._constraints = constraints
         self.platform.constraints = constraints
 
+    def _update_state_descriptor(self, descriptor: StateDescriptor, *args, **kwargs):
+        """
+         Called on execution_intermittent to update the descriptor for this state. This is intended for information
+         like the PC or instruction count, where updating after each instruction would be a waste of cycles.
+         This one updates the execution counts
+         :param descriptor: StateDescriptor for this state
```

Comment:
I think a newline is required for docs
```suggestion

         :param descriptor: StateDescriptor for this state
```